client/lib/cart-values/cart-items.js

```diff
@@ -54,97 +46,20 @@ import {
 import { getTld } from 'calypso/lib/domains';
 import { domainProductSlugs } from 'calypso/lib/domains/constants';
 
-/**
- * @typedef { import("./types").CartItemValue} CartItemValue
- * @typedef { import("./types").CartItemExtra} CartItemExtra
- * @typedef { import("./types").CartValue } CartValue
- */
-
-/**
- * Adds the specified item to a shopping cart.
- *
- * @param {CartItemValue} newCartItem - new item as `CartItemValue` object
- * @returns {Function} the function that adds the item to a shopping cart
- */
-export function addCartItem( newCartItem ) {
-	function appendItem( products ) {
-		products = products || [];
-
-		const isDuplicate = products.some( function ( existingCartItem ) {
-			return isEqual( newCartItem, existingCartItem );
-		} );
-
-		return isDuplicate ? products : products.concat( [ newCartItem ] );
-	}
-
-	return function ( cart ) {
-		if ( cartItemShouldReplaceCart( newCartItem, cart ) ) {
-			return update( cart, { products: { $set: [ newCartItem ] } } );
-		}
-
-		return update( cart, { products: { $apply: appendItem } } );
-	};
-}
-
-/**
- * Determines if the given cart item should replace the cart.
- * This can happen if the given item:
- * - will result in mixed renewals/non-renewals or multiple renewals (excluding privacy protection).
- *
- * @param {CartItemValue} cartItem - `CartItemValue` object
- * @param {object} cart - the existing shopping cart
- * @returns {boolean} whether or not the item should replace the cart
- */
-export function cartItemShouldReplaceCart( cartItem, cart ) {
```

Comment:
There's a [comment reference to this function here](https://github.com/Automattic/wp-calypso/blob/remove%2Funused-cart-values-functions/packages/shopping-cart/src/cart-functions.ts#L206).

Reply:
That's a good idea. I updated the comment.